app/lib/rocketchat/services/sdk.ts

```diff
@@ -0,0 +1,30 @@
+import { Rocketchat as RocketchatClient } from '@rocket.chat/sdk';
+
+import reduxStore from '../../createStore';
+import { useSsl } from '../../../utils/url';
+
+class Sdk {
+	private sdk: any;
```

Comment:
You can do as I did here with rocket, if the sdk is a method lib.
https://github.com/RocketChat/Rocket.Chat.ReactNative/blob/dce6c3ffb6200341d01b8d66804957d32cebc413/app/definitions/IRocketChat.ts